chemprop/train/evaluate.py

```diff
@@ -2,20 +2,24 @@
 import logging
 from typing import Dict, List
 
+import numpy as np
+
 from .predict import predict
 from chemprop.data import MoleculeDataLoader, StandardScaler
 from chemprop.models import MoleculeModel
 from chemprop.train import get_metric_func
 
 
-def evaluate_predictions(preds: List[List[float]],
-                         targets: List[List[float]],
-                         num_tasks: int,
-                         metrics: List[str],
-                         dataset_type: str,
-                         gt_targets: List[List[bool]] = None,
-                         lt_targets: List[List[bool]] = None,
-                         logger: logging.Logger = None) -> Dict[str, List[float]]:
+def evaluate_predictions(
+    preds: List[List[float]],
+    targets: List[List[float]],
+    num_tasks: int,
```

Comment:
is this argument still necessary? should be inferable from the shape of both `preds` and `targets`